src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -53,6 +55,18 @@ public DefaultTargetService(ClientConfig clientConfig, TelemetryService telemetr
     } else {
       this.targetHttpClient = targetHttpClient;
     }
+
+    Properties defaultProps = new Properties();
+    try {
+      InputStream in = getClass().getResourceAsStream("/gradle.properties");
+      BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+      defaultProps.load(reader);
+      in.close();
+    } catch (IOException e) {
+      System.out.println("unable to location gradle.properties file");
```

Comment:
Let's make the error message more indicative of the feature that is failing - since this will be output to customers' servers, i.e. "Unable to determine Target SDK version"